docs/language-basics/documentation.md

```diff
@@ -0,0 +1,74 @@
+---
+id: documentation
+title: Documentation
+---
+
+# Documentation
+
+Documentation is crucial to any API. TypeSpec provides a number of ways to document your API using TSDoc doc comments and decorators.
+
+# Comments
+
+TypeSpec supports both single-line and multi-line comments. Single-line comments start with `//` and continue until the end of the line. Multi-line comments start with `/*` and continue until the closing `*/` is encountered.
+
+```typespec
+// This is a single-line comment
+model Dog {
+  /* This is a multi-line comment
+  that spans multiple lines */
+  name: string;
+}
+```
+
+Comments are ignored by the compiler and are not included in the generated output. They are intended to be used to document your spec internally and are not suitable for generating external documentation.
```

Comment:
Because of the ordering of topics, this seems a bit misleading.  I would consider putting the 'Comments' section *after* the section on Documenting APIs.  If not that, then a small sentence here that TSDoc comments can be used to document the api and may appear in generated artifacts